nautobot/extras/tests/test_datasources.py

```diff
@@ -217,6 +248,19 @@ def populate_repo(path, url):
                     {"ntp-servers": ["172.16.10.22", "172.16.10.33"]},
                     config_context.data,
                 )
+                self.assertEqual(self.config_context_schema["_metadata"]["name"], config_context.schema.name)
+
+                # Make sure ConfigContextSchema was successfully loaded from file
+                config_context_schema_record = ConfigContextSchema.objects.get(
+                    name="Config Context Schema 1",
+                    owner_object_id=self.repo.pk,
+                    owner_content_type=ContentType.objects.get_for_model(GitRepository),
+                )
+                config_context_schema = self.config_context_schema
+                config_context_schema_metadata = config_context_schema.setdefault("_metadata", {})
```

Comment:
`setdefault` here seems unnecessary and potentially misleading.
```suggestion
                config_context_schema_metadata = config_context_schema["_metadata"]
```